src/main.rs

```diff
@@ -47,64 +49,100 @@ enum Command {
     InstallHooks,
 }
 
-fn main() -> Result<(), Box<dyn Error>> {
-
+// This function intentionally doesn't return 'error', it's meant to
+// nicely log any errors that happened further down and, if there are
+// errors, exit with a non-zero code.
+fn main() {
     let opt = Opt::from_args();
+
     stderrlog::new()
         .module(module_path!())
         .quiet(opt.quiet)
-        .verbosity(opt.verbose + 2) // Default is info
-        .init()?;
+        .verbosity(opt.verbose + 2) // Default is info (2)
+        .init()
+        .expect("ERROR: Logger was initialized twice");
     
     info!("{}", block("Ahoy, maties! Welcome to Capn Githook!"));
 
-    info!("Read configuration file");
-    let config = match LiveGit::new()?.read_config() {
+    let git = match LiveGit::new() {
+        Ok(g) => g,
+        Err(e) => {
+            error!("Capn Githook must be called in a Git repo. Error: {}", e);
```

Comment:
Log the error first and then make the suggestion that it should be called from inside of a git repo.